cmd/generator/add.go

```diff
@@ -15,6 +15,10 @@ import (
 
 func init() {
 	generatorCmd.AddCommand(addCmd)
+
+	addCmd.Flags().Bool("beta", false, "Mark release as Beta")
+	addCmd.Flags().Bool("official", false, "Mark this plugin is maintanied by Mattermost")
```

Comment:
```suggestion
	addCmd.Flags().Bool("official", false, "Mark this plugin as maintained by Mattermost")
```

Reply:
I requested the review to soon :confused: